routes/api/v1/foods.js

```diff
@@ -37,4 +34,42 @@ router.get("/:id", function(req, res, next) {
   });
 });
 
+router.post("/", function(req, res, next) {
+  res.setHeader("Content-Type", "application/json");
+  Food.findOrCreate({where: {name: req.body.name, calories: req.body.calories} })
+  .then(food => {
+    if (!food){
+      res.status(400).send({ error })
+    } else {
+      res.status(200).send(JSON.stringify(parsedFood(food[0])))
+    }
+  })
+  .catch(error => {
+    res.status(500).send({ error })
+  })
+})
+
+function parsedFood(food) {
```

Comment:
This looks nice. Pretty clean so probably not worth refactoring into a serializer or something.